ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml

```diff
@@ -248,6 +215,79 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
+  cron: 0 8 */2 * *
+  decorate: true
+  extra_refs:
+  - base_ref: main
+    org: redhat-chaos
+    repo: krkn-hub
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale-qe
+    ci-operator.openshift.io/variant: arm
+    ci.openshift.io/generator: prowgen
+    job-release: "4.13"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm
+  spec:
```

Comment:
let's add the slack integration into the new tests we added

Reply:
Done!